cellprofiler/modules/measureobjectneighbors.py

```diff
@@ -94,96 +97,106 @@ class MeasureObjectNeighbors(cpm.Module):
+                'Neighbor distance', 5, 1, doc="""\
+*(Used only when “%(D_WITHIN)s” is selected)*
+
+The Neighbor distance is the number of pixels that each object is
+expanded for the neighbor calculation. Expanded objects that touch are
+considered neighbors.
+""" % globals())
 
         self.wants_count_image = cps.Binary(
                 'Retain the image of objects colored by numbers of neighbors?',
-                False, doc="""
-            An output image showing the input objects
-            colored by numbers of neighbors may be retained. A colormap of your choice shows
-            how many neighbors each object has. The background is set
-            to -1. Objects are colored with an increasing color value
-            corresponding to the number of neighbors, such that objects with no
-            neighbors are given a color corresponding to 0. Use the <b>SaveImages</b>
-            module to save this image to a file.""")
+                False, doc="""\
+An output image showing the input objects colored by numbers of
+neighbors may be retained. A colormap of your choice shows how many
+neighbors each object has. The background is set to -1. Objects are
+colored with an increasing color value corresponding to the number of
+neighbors, such that objects with no neighbors are given a color
+corresponding to 0. Use the **SaveImages** module to save this image to
+a file.""")
 
         self.count_image_name = cps.ImageNameProvider(
                 'Name the output image',
-                'ObjectNeighborCount', doc="""
-            <i>(Used only if the image of objects colored by numbers of neighbors
-            is to be retained for later use in the pipeline)</i> <br>
-            Specify a name
-            that will allow the image of objects colored by numbers of neighbors
-            to be selected later in the pipeline.""")
+                'ObjectNeighborCount', doc="""\
+*(Used only if the image of objects colored by numbers of neighbors is
+to be retained for later use in the pipeline)*
+
+Specify a name that will allow the image of objects colored by numbers
+of neighbors to be selected later in the pipeline.""")
 
         self.count_colormap = cps.Colormap(
-                'Select colormap', doc="""
-            <i>(Used only if the image of objects colored by numbers of neighbors
-            is to be retained for later use in the pipeline)</i> <br>
-            Select the colormap to use to color the neighbor number image. All available colormaps can be seen
-            <a href="http://www.scipy.org/Cookbook/Matplotlib/Show_colormaps">here</a>.""")
+                'Select colormap', doc="""\
+*(Used only if the image of objects colored by numbers of neighbors is
+to be retained for later use in the pipeline)*
+
+Select the colormap to use to color the neighbor number image. All
+available colormaps can be seen `here`_.
+
+.. _here: http://www.scipy.org/Cookbook/Matplotlib/Show_colormaps""")
```

Comment:
This link is out of date, please use http://matplotlib.org/examples/color/colormaps_reference.html